### DISCUSSION TITLE
Can Next.js Export Static Site in Parts?

### DISCUSSION THREAD

**Discussion by tribe56123** [vercel/next.js#40882] [Help]
I am using Next.js's Static HTML Export for my site which have 10 million static pages but I am running into ram issues when building the app.

Is it even possible to export it in parts like 100k pages on first build then 100k on second build?

I do not want to use Incremental Static Regeneration or getServerSideProps to cut costs of Node.js server and database.

**Comment by wilsonneto-swe** [comment]
AFAIK we don`t have a way to solve this problem...

I guess you can do something by editing the exportMap in each build and so joining the pages together by uploading them in each build, but I have never seen something similar, all the time I faced this kind of problem, I have used the ISG approach...
